editor/editor_file_system.h

```diff
@@ -214,9 +217,11 @@ class EditorFileSystem : public Node {
 
 	struct ImportFile {
 		String path;
+		String importer;
+		bool threaded;
```

Comment:
This variable is used few times, but I didn't see any initialization

Reply:
It should be always initialized when used, but I will default it to false anyway.